string robot_name
-------
string state